package.json

```diff
@@ -43,10 +45,12 @@
     "flexboxgrid": "^6.3.1",
     "flexboxgrid-helpers": "^1.1.3",
     "js-slang": "^0.2.3",
+    "jwt-decode": "^2.2.0",
     "lodash": "^4.17.11",
     "lz-string": "^1.4.4",
     "moment": "^2.22.2",
-    "node-sass-chokidar": "^1.3.4",
+    "node-sass": "^4.12.0",
```

Comment:
I have to install `node-sass` every time I clone the frontend. Otherwise all the imports in `index.scss` can't be compiled to css.

Not sure if this just happens to my computer though.

Reply:
@wardetu what happens if your try on our current master branch? Do you need to install `node-sass` also? Or is it only when you are working on your fork for this PR.

Reply:
I always have to install. Prolly this is because I'm on Windows.

Reply:
Try removing it and then doing a clean reinstall

Reply:
It worked.
I have updated `package.json` accordingly.

Reply:
Whenever you update dependencies, please commit the `package-lock` as well

Reply:
Okay!